src/gretel_trainer/models.py

```diff
@@ -0,0 +1,107 @@
+import logging
+from typing import Union
+
+from gretel_client.projects.models import read_model_config
+
+
+class _BaseConfig:
+    """This class should not be used directly, models should be
+    derived from this class
+    """
+
+    _max_header_clusters: int
+    """This should be overridden on concrete classes"""
+
+    # Default values for all configs, may be overridden if needed
+    _max_rows: int = 1000000
+    _enable_privacy_filters: bool = False
```

Comment:
This is not used. Should it be used somewhere? Or can it be deleted?
Or maybe it shouldn't have the `_`?

Reply:
These can be deleted